OpenMage/magento-lts
# Fix Deprecated Functionality Error Caused by Passing NULL to nl2br() in Sharing Template

### Description (*)
Prevents an error caused by passing null to nl2br() in the rwd default catalog product view sharing template if the product short description is not set.

It is possible for the short description product attribute to be empty if a site is upgraded from legacy Magento and/or the attribute setting is changed to Values Required "No".

If the product short description isn't set, the sharing template passes null to nl2br(), resulting in "ERR (3): Deprecated functionality: nl2br(): Passing null to parameter #1 ($string) of type string is deprecated in [docroot]/app/design/frontend/rwd/default/template/catalog/product/view/sharing.phtml on line 30".

This change makes sure even if the product short description is null, we at least pass an empty string to nl2br().

### Related Pull Requests
n/a

### Fixed Issues (if relevant)
n/a

### Manual testing scenarios (*)
1. If it isn't already, change the short description product attribute Values Required "No"
2. Create a product with no short description or remove it from an existing one
3. Browse the catalog product view and see that nl2br throws an error in the system log

### Questions or comments
- How do the automated tests mentioned in the contribution checklist work, and how can I tell if all builds are green please?
- I'm new here; am I doing this right?
- Thank you all so much for your dedication this project!

### Contribution checklist (*)
